app/scripts/modules/core/src/managed/overview/EnvironmentsOverview.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { Application } from 'core/index';
+
+import { CollapsibleSection } from './CollapsibleSection';
+import { Resource } from './Resource';
+import { Artifact } from './artifact/Artifact';
+import { useFetchApplicationQuery } from '../graphql/graphql-sdk';
+import { QueryEnvironment } from './types';
+
+import './EnvironmentsOverview.less';
+import './baseStyles.less';
+
+interface IEnvironmentsProps {
+  app: Application;
+}
+
+export const EnvironmentsOverview = ({ app }: IEnvironmentsProps) => {
+  const { data, error } = useFetchApplicationQuery({ variables: { appName: app.name } });
+  if (error) {
+    // TODO: notify users
+    console.warn(error);
```

Comment:
to toast, or not to toast?

Reply:
Nah, I think that showing it in the UI is better